_posts/2019-10-28-MongoDB.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: post
+current: post
+cover: mongodb-article-cover.jpg
+title: How to Securely Configure MongoDB
+date: 2019-11-24 09:00:00
+category: tutorials
+author: Mahmoud Fahmy
+tags: [tutorials, Open Source]
```

Comment:
Double check this with older posts. They need to have the same capitalization and I don't know if you can have spaces (should likely be 'open-source').